SEED-platform/seed
# (2175) Feat/api v3 import files

#### What's this PR do?
Bump up the `import_files` endpoint to v3, tweak some names, and make swagger docs. Also, removed a dead endpoint, `data_quality_progress`.

#### How should this be manually tested?
Test each of the endpoints with `/import_files/` prefix
- probably a good idea to import at least two files for testing (one for deleting and another for running the other endpoints on)

Also test `organizations/{id}/column_mappings` endpoint.
- Just fill the template data with something reasonable, then you can check in the shell with `ImportFile.objects.get(id=<your id here>).cached_mapped_columns`
#### What are the relevant tickets?
#2175
